resolve.js

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env node
+
+'use strict';
+
+var resolve = require('json-refs').resolveRefs;
+var YAML = require('js-yaml');
+var fs = require('fs');
+
+var program = require('commander');
+
+program
+  .version('2.0.0')
+  .option('-o --output-format [output]',
+          'output format. Choices are "json" and "yaml" (Default is json)',
+          'json')
+  .usage('[options] <yaml file ...>')
+  .parse(process.argv);
+
+if (program.outputFormat !== 'json' && program.outputFormat !== 'yaml') {
+  console.error(program.help());
+  process.exit(1);
+}
+
+var file = program.args[0];
+
+if (!fs.existsSync(file)) {
+  console.error('File does not exist. ('+file+')');
+  process.exit(1);
+}
+
+var root = YAML.safeLoad(fs.readFileSync(file).toString());
+var options = {
+  filter        : ['relative', 'remote'],
```

Comment:
maybe filter options should be not fixed here but passed as optional parameters via command line?

Reply:
I had planned to remove `resolve.js` since it's technically `multi-file-swagger` itself.
If we plan to edit `multi-file-swagger` we can PR our changes to the packages repo itself.